build/debian-hyperkube-base/Makefile

```diff
@@ -19,7 +19,7 @@
 
 REGISTRY?=gcr.io/google-containers
 IMAGE?=debian-hyperkube-base
-TAG=0.8
+TAG=0.10
```

Comment:
this is a bit misleading, since it doesn't also cherrypick in https://github.com/kubernetes/kubernetes/commit/6fd8903af354c6c6d4d479102132b7cdd5451665 which adds `ipset`.

can we maybe drop all changes to `build/debian-hyperkube-base` and only update the references (`build/root/WORKSPACE` and `cluster/images/hyperkube/Makefile`)?